lib/cocoapods/project.rb

```diff
@@ -53,7 +53,7 @@ def initialize(path, skip_initialization = false,
       @development_pods = new_group('Development Pods')
       @dependencies_group = new_group('Dependencies')
       @pod_target_subproject = pod_target_subproject
-      @project_name = Pathname(path).basename('.*')
+      @project_name = Pathname(path).basename('.*').to_s
```

Comment:
can you explain a bit more about this change?

Reply:
`project_name` is defined on `Project` as a String, but it was actually a `Pathname`. I could have updated the doc string but it seems more correct to have a name be a `String`

https://github.com/CocoaPods/CocoaPods/blob/02be85c8b83a1b45f7a72cd568d5f3b02bf60f41/lib/cocoapods/project.rb#L35-L37

Reply:
thanks!